Tools/LambdaTestTool/src/Amazon.Lambda.TestTool.BlazorTester/Pages/Runtime.razor

```diff
@@ -0,0 +1,297 @@
+@page "/runtime"
+
+@using Amazon.Lambda.TestTool.BlazorTester.Services
+@using Amazon.Lambda.TestTool.SampleRequests;
+@using Microsoft.AspNetCore.Http;
+
+@inject IHttpContextAccessor httpContextAccessor;
+@inject LocalLambdaOptions LambdaOptions
+@inject IRuntimeApiDataStore RuntimeApiModel
+@inject IModalService Modal
+
+<h2>Test Executable Assembly</h2>
+
+<p>
+    For Lambda functions written as executable assemblies, i.e. custom runtimes functions and top level statement functions, this page can be used for testing the functions locally. 
+    Set the <b>AWS_LAMBDA_RUNTIME_API</b> environment variable to <b>localhost:@httpContextAccessor.HttpContext.Connection.LocalPort</b> while debugging executable assembly 
+    Lambda function. More information can be found in the <a href="documentation">documentation</a>.
```

Comment:
Not sure this is an error but for consistency should this be /documentation or should the one in index.razor be `runtime`?

Reply:
Change to `/documentation`